src/ApiTemplates/TMDBApiCaller.js

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+var config = require("../config");
+
+export default async (pageNumber) => {
+  const tmdbApiKey = config.TMDB.tmdbApiKey;
+  const expressServerTMDBApiUrl = `https://api.themoviedb.org/3/trending/movie/day?api_key=${tmdbApiKey}&page=${pageNumber}`;
+
+  try {
+    const tmdbApiData = await axios.get(expressServerTMDBApiUrl);
+    const arrayOfMovies = tmdbApiData.data.results;
+    return {
+      listOfTrendingFilms: arrayOfMovies,
+    };
+  } catch (error) {
```

Comment:
maybe want to catch this error higher up the stack so that you can put a message in the ui or something?

Reply:
With regards to this, I am in a dilema
So I added this as a temporary call to server. This whole call will be added to my [repo](https://github.com/RonakSharma1/funtown-web-application-express-server) which deals with calls to server. Therefore, I think it might not be worth making this efficient here. Would you disagree?